numba/cuda/cudadrv/devicearray.py

```diff
@@ -119,14 +134,9 @@ def __cuda_array_interface__(self):
         else:
             ptr = 0
 
-        if array_core(self).flags['C_CONTIGUOUS']:
```

Comment:
FWIW I think we could also just do `self.flags['C_CONTIGUOUS']`, which doesn't appear to use a CUDA context.

Reply:
`self.flags['C_CONTIGUOUS']` is `false` on broadcasted views, which is why the original code use `array_core(self)`. `is_contiguous()` now also ignores broadcasted dimensions.